packages/@azure/servicebus/data-plane/package.json

```diff
@@ -76,7 +76,7 @@
     "build-test": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js",
     "build-samples": "cd examples && tsc -p .",
     "test": "npm run build",
-    "unit": "npm run build-test && mocha -t 50000 test-dist/index.js",
+    "unit": "npm run build-test && mocha -t 450000 test-dist/index.js",
```

Comment:
I found that we can change the timeout for individual tests. Therefore instead of changing the timeout here which will affect all tests, lets just change for the tests that need the higher timeout.

The way to do this is to call `.timeout(45000)` at the end of the `it` or `describe` call